__tests__/endpoints/topics.test.js

```diff
@@ -10,7 +10,12 @@ describe("/api/topics", () => {
 			.expect(200)
 			.then(({ body: { topics } }) => {
 				expect(topics.length).toBe(3);
-				expect(topics).toMatchObject(data.topicData);
+				topics.forEach((topic) => {
+					expect(topic).toHaveProperty("slug")
```

Comment:
This assertion is implicit in the next assertion. The same with the other similar assertions here